src/contractor/__main__.py

```diff
@@ -127,21 +133,34 @@ def deploy(ctx, config, community, network, keyfile, password, chain, db_uri, gi
               help='Is this deployment on the homechain or sidechain?')
 @click.option('--token', envvar='TOKEN', required=True, type=click.Choice(('nectar', 'ether')),
               help='Which token balance you want to monitor')
-@click.option('--verbosity', type=click.Choice((1, 2)), envvar='VERBOSITY', required=False,
-              help='1 for minimum logs or 2 all the log block, tx, and function input logs', default=1)
-@click.option('--cumulative', envvar='CUMULATIVE', is_flag=True, required=False,
-              help='If balance change and function call counts should be cumulatively added or compared to the last block', default=False)
+@click.option('-v', '--verbose', count=True,
+              help='Verbosity level')
+@click.option('--cumulative', envvar='CUMULATIVE', is_flag=True,
+              help='Cumulatively track balance change and function call counts')
+@click.option('-a', '--artifactdir', type=click.Path(exists=True, file_okay=False), default='build',
+              help='Directory containing the compiled artifacts to deploy')
+@click.option('-i', '--input', type=click.Path(exists=True, dir_okay=False), required=False,
+              help='Input file containing the deployed addresses of our artifacts')
 @click.pass_context
-def watch(ctx, config, community, network, chain, token, verbosity, cumulative):
+def watch(ctx, config, community, network, chain, token, verbosity, cumulative, artifactdir, input):
```

Comment:
```suggestion
def watch(ctx, config, community, network, chain, token, verbose, cumulative, artifactdir, input):
```

Name doesn't match the option